README.md

```diff
@@ -2,7 +2,8 @@
 [![pub package](https://img.shields.io/pub/v/web.svg)](https://pub.dev/packages/web)
 [![package publisher](https://img.shields.io/pub/publisher/web.svg)](https://pub.dev/packages/web/publisher)
 
-Lightweight browser API bindings built around JS static interop.
+Lightweight browser API bindings built around
```

Comment:
We may want to update the package description as well: https://github.com/dart-lang/web/blob/main/pubspec.yaml#L3.